js/repl/UriUtils.js

```diff
@@ -84,7 +86,7 @@ const updateQuery = (state: PersistedState) => {
       return null;
     } else if (key === "code") {
       return `${key}_lz=` + compress(state.code);
-    } else {
+    } else if(state[key] !== URL_DEFAULTS[key]){
```

Comment:
This isn't quite right. The way the REPL is setup is this:
* If a flag is specified in the URL, we use its value
* Else we check `localStorage` for the most recently-used value and use it,
* Else we fallback to a (hard-coded) default

Reply:
If folks feel that the convenience of using localStorage to remember last-used values isn't important, then we could drop it and move forward with this change.

Alternately we could change the REPL to only use localStorage if there's a special flag in the URL (which we could pass in from the REPL link on the website). That's the most useful use case of it really.

Maybe it's better to only read/write localStorage when we have explicit permission anyway? This would prevent a shared link from overriding our last explicitly chosen settings?

Reply:
FWIW, I'd like to keep localstorage... just need to iron out a few bugs and I think it's fine.

Reply:
Bugs with our localStorage use? Hm. I wasn't aware of any. :)

Reply:
@bvaughn @existentialism Thanks for the clarification. So I'm gonna Make the priority in this order **URL > LOCAL STORAGE > DEFAULTS**.

I'll keep posted once the change is done.

Reply:
@rajzshkr 👍